site/docs/flink.md

```diff
@@ -28,13 +28,14 @@ we only integrate iceberg with apache flink 1.11.x .
 | [SQL create table like](#create-table-like)                            | ✔️                 |                                                        |
 | [SQL alter table](#alter-table)                                        | ✔️                 | Only support altering table properties, Columns/PartitionKey changes are not supported now|
 | [SQL drop_table](#drop-table)                                          | ✔️                 |                                                        |
-| [SQL select](#querying-with-sql)                                       | ✔️                 | Only support batch mode now.                           |
+| [SQL select](#querying-with-sql)                                       | ✔️                 | Support both streaming and batch mode                  |
 | [SQL insert into](#insert-into)                                        | ✔️ ️               | Support both streaming and batch mode                  |
 | [SQL insert overwrite](#insert-overwrite)                              | ✔️ ️               |                                                        |
 | [DataStream read](#reading-with-datastream)                            | ✔️ ️               |                                                        |
 | [DataStream append](#appending-data)                                   | ✔️ ️               |                                                        |
 | [DataStream overwrite](#overwrite-data)                                | ✔️ ️               |                                                        |
-| [Metadata tables](#inspecting-tables)                                  |    ️               |                                                        |
+| [Metadata tables](#inspecting-tables)                                  |    ️               | Support Java API but does not support Flink SQL        |
+| [Rewrite files action](#rewrite-files-action)                          | ✔️ ️               |                                                        |
```

Comment:
should we add a 1.12 column?

Reply:
I don't think that support currently works in 1.12, according to the docs?

Reply:
Yeah, that's why I am asking if we should add a column to suggest which ones work for 1.12.